lib/tpl/dokuwiki/css/_admin.less

```diff
@@ -0,0 +1,59 @@
+.dokuwiki div.UI-admin {
+
+    /* main task grouped in two columns */
+    ul.admin_tasks {
+        float: left;
+        width: 40%;
+        list-style-type: none;
+        font-size: 1.125em;
+    }
+    [dir=rtl] ul.admin_tasks {
+        float: right;
+    }
+
+    /* general menu item styling */
+    ul li {
+        margin: 0 0 1em 0;
+        font-weight: bold;
+        list-style-type: none;
+
+        div {
+            // flexbox takes care of RTL alignment
```

Comment:
The icon automatically goes to the right when using RTL because it is inline-block by default. You don't need flexbox for that. It would make more sense to use flexbox for the two uls for that reason than the icon. (Although I don't think it would be worth it as the extra CSS for RTL is only one line.)

I personally don't like the vertically centred icon on items with two lines. It disturbs the horizontal rhythm. But in both cases the current wonkiness improves by adding `vertical-align: bottom` to the `svg`.